### ISSUE TITLE
Entry transition not working when floating-label=true

### ISSUE THREAD

**Issue by pedrito-96** [formkit/formkit#1212]
### Reproduction

https://formkit.link/4472a63ff5f2916621a2a2e09e99eebb

### Describe the bug

The vue transition in entrance is not reproduced when floating-label is active.
To reproduce the problem in the playground linked above you just need to refresh the page or submit the form.

In my project I use Formkit v.1.5.9 and Nuxt 3.10.2 with Nitro 2.8.1
In my case, with a form inside a side slide component with visibility toggled by a useState, the entrance transition never works, only the exit one (when  floating-label is set to true ).
In the reproduction above, it works for the attempts after the first.

### Environment

• OS: macOS 14 Sonoma
• Browser: chrome, firefox, safari

**Comment by Archetipo95** [comment]
Here you can see it in action:
https://github.com/formkit/formkit/assets/11798239/bd764d08-b692-4b4a-8bf0-2a4c00111b1b

**Comment by andrew-boyd** [comment]
This is wild... looking into it.

**Comment by andrew-boyd** [comment]
Ok, I _think_ we got to the bottom of this one.

TL;DR - this may be a byproduct of using Tailwind via the CDN and how their JIT compiler works — which our Playground does and I'm hoping the current version of your Nuxt app does too.

---

So we set up a new Nuxt application and were unable to get this behavior to show itself using the same example that you posted into Playground. Here you can see that in a standard Nuxt app the animation works just fine:

https://stackblitz.com/edit/nuxt-starter-cfiycv?file=app.vue

We were racking our brains over here trying to figure out what may be different about this setup from both your Nuxt app you allude to in your post but also in common with the FormKit Playground. The only guess we can come up with is that Tailwind is being loaded via the CDN — Something that we need for the Playground because the app is runtime only. If we use the CDN version of Tailwind in the same Nuxt app you can see that the issue shows up again.

https://stackblitz.com/edit/nuxt-starter-crvtjp?file=pages%2Findex.vue

So — the million dollar question — are you indeed using Tailwind via the CDN in your application? If you are then switching to a build-time version of Tailwind should get everything working for you. If not then... well I suppose I have a lot more questions and no answers at the moment. 😅

**Comment by Archetipo95** [comment]
Unfortunately we are not using tailwind with the CDN, I will try to make a POC as soon as I have some free time, thank you for your time!

**Comment by andrew-boyd** [comment]
Hmmmmm… the mystery deepens. Please do put together a repo when you have the time — I really want to figure out what may be causing this. 👀

**Comment by Archetipo95** [comment]
Here it is! https://stackblitz.com/~/github.com/Archetipo95/formkit-poc-broken-animation
I don't know what we are doing is wrong but it could be something with this part of the code 
`<style lang="scss" src="@design-system/assets/css/floatingLabelFormkit.scss" />`

**Comment by Archetipo95** [comment]
So I improved the POC and cleaned the code. I'm noticing that the animation is broken just with the HeadlessUI component when floating label is active.

Repository: https://github.com/Archetipo95/formkit-poc-broken-animation
Site: https://main--formkit-experiments.netlify.app

**Comment by andrew-boyd** [comment]
Thank you for this reproduction — I'll look into it.

**Comment by andrew-boyd** [comment]
@Archetipo95 I have a fix for this that will be released in `1.6.1`
